generic/utils/ras.py.data/common_functions.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2016 IBM
+# Author: Pavithra <pavrampu@linux.vnet.ibm.com>
```

Comment:
In the previous PR this file was authored by @lmr. I can see it differs a bit, but I really don't know how much. So I need to ask you, did you re-implement this from scratch or did you use the previously existing files? You need to use the files accordingly to the license, so if it's compatible you can use it, but you need to give credits to the author. And depending on the license you might not have the permission to modify it or even use it here. Do you have any links to the original code? Was it already upstream somewhere? I only saw Lucas's name in the previous PR but most of the scripts were without an author.